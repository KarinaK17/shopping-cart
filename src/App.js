import "./App.css";
import React from "react";
import { HashRouter, Routes, Route } from "react-router-dom";
import Nav from "./components/Nav";
import Home from "./components/Home";
import Shop from "./components/Shop";

function App() {
  return (
    <React.Fragment>
      <HashRouter>
        <Nav />
        <Routes>
          <Route path="/" element={<Home />} />
          <Route path="/shop" element={<Shop />} />
        </Routes>
        <footer>
          Images from <a href="https://darvish.by/">Darvish</a>,{" "}
          <a href="https://www.svgrepo.com/">SVG Repo</a>,{" "}
          <a href="https://pixabay.com/">Pixabay</a>
        </footer>
      </HashRouter>
    </React.Fragment>
  );
}

export default App;
