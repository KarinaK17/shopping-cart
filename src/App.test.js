import { render, screen } from "@testing-library/react";
import React from "react";
import userEvent from "@testing-library/user-event";
// import Nav from "./components/Nav";
// import Home from "./components/Home";
// import Shop from "./components/Shop";
import App from "./App";

describe("App component", () => {
  it("renders correctly", () => {
    const { container } = render(<App />);
    expect(container).toMatchSnapshot();
  });
});
