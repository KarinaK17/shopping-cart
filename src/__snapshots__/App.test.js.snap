// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders correctly 1`] = `
<div>
  <nav>
    <a
      class="logo-and-name"
      href="#/"
    >
      <img
        alt="logo"
        class="logo"
        src="logo.svg"
      />
      <h1>
        Ã‰tude
      </h1>
    </a>
    <ul>
      <a
        aria-current="page"
        class="active"
        href="#/"
      >
        <li>
          Home
        </li>
      </a>
      <a
        class=""
        href="#/shop"
      >
        <li>
          Shop
        </li>
      </a>
    </ul>
  </nav>
  <div
    class="content Home"
  >
    <div
      class="welcome"
    >
      <h2>
        Home Page
      </h2>
      <p>
        Welcome to our store!
      </p>
    </div>
    <div
      class="about"
    >
      <h3>
        About us
      </h3>
      <p>
        We sell high quality stationery, school and office supplies to make your studying, working or making art comfortable and fun.
      </p>
    </div>
    <div
      class="home-images"
    >
      <img
        alt="kids"
        class="kids"
        src="kids.png"
      />
      <img
        alt="girl"
        class="girl"
        src="girl.png"
      />
    </div>
    <div
      class="contacts"
    >
      <h3>
        Contacts
      </h3>
      <p>
        Telephone: +307 (24) 53-98-01
      </p>
    </div>
  </div>
  <footer>
    Images from 
    <a
      href="https://darvish.by/"
    >
      Darvish
    </a>
    ,
     
    <a
      href="https://www.svgrepo.com/"
    >
      SVG Repo
    </a>
    ,
     
    <a
      href="https://pixabay.com/"
    >
      Pixabay
    </a>
  </footer>
</div>
`;
